server/gitrest/packages/gitrest-base/src/runnerFactory.ts

```diff
@@ -58,18 +70,23 @@ export class GitrestResourcesFactory implements core.IResourcesFactory<GitrestRe
 		);
 	}
 
-	private getFileSystemManagerFactories(config: Provider): IFileSystemManagerFactories {
+	private getFileSystemManagerFactories(
+		config: Provider,
+		customizations?: IGitrestResourcesCustomizations,
+	): IFileSystemManagerFactories {
 		const defaultFileSystemName: string = config.get("git:filesystem:name") ?? "nodeFs";
 		const ephemeralFileSystemName: string =
 			config.get("git:ephemeralfilesystem:name") ?? "redisFs";
 
 		const defaultFileSystemManagerFactory = this.getFileSystemManagerFactoryByName(
 			defaultFileSystemName,
 			config,
+			customizations?.redisClientConnectionManagerForDefaultFileSystem,
```

Comment:
nit: maybe pass customizations to this private method instead of redisclient. You can fetch that from customizations inside the method.

Reply:
Oh I think the NodeFsFileSystem does not use a RedisClient. I can get rid of the customization `redisClientConnectionManagerForDefaultFileSystem`. Thanks for catching this!

Reply:
Based on our conversation about how there could be a case where the `defaultFileSystem` is also an instance of `RedisFs`, I will not change this code.